Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj

```diff
@@ -31,18 +31,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac" Version="5.1.2" />
-    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="6.0.0" />
-    <PackageReference Include="FluentValidation" Version="8.6.2" />
+    <PackageReference Include="Autofac" Version="6.0.0" />
```

Comment:
@jamessmckay In the JIRA ticket you wrote there should be no major version upgrades. Does this constitute a major version upgrade?

Reply:
I limited to  minor upgrades . thanks